package.json

```diff
@@ -32,16 +30,24 @@
   },
   "homepage": "https://github.com/indutny/elliptic",
   "devDependencies": {
-    "coveralls": "^2.11.3",
-    "istanbul": "^0.4.2",
-    "jscs": "^2.9.0",
-    "jshint": "^2.6.0",
-    "mocha": "^2.1.0"
+    "brfs":  "1.4.3",
+    "coveralls": "2.11.3",
+    "grunt": "0.4.5",
+    "grunt-browserify": "5.0.0",
+    "grunt-contrib-connect": "1.0.0",
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-contrib-uglify": "1.0.1",
+    "grunt-mocha-istanbul": "3.0.1",
+    "grunt-saucelabs": "8.6.2",
+    "istanbul": "0.4.2",
+    "jscs": "2.9.0",
+    "jshint": "2.6.0",
+    "mocha": "2.1.0"
   },
   "dependencies": {
-    "bn.js": "^4.0.0",
-    "brorand": "^1.0.1",
-    "hash.js": "^1.0.0",
-    "inherits": "^2.0.1"
+    "bn.js": "4.0.0",
+    "brorand": "1.0.1",
+    "hash.js": "1.0.0",
+    "inherits": "2.0.1"
```

Comment:
@evilaliv3 while I understand your concern about versioning, I would prefer to keep `^`, because it is natural for semver, and I am in charge of many of these modules ;)